ush/get_data_files.py

```diff
@@ -394,6 +394,8 @@ def get_hpss_data(hpss_job_filename, save_data_dir, save_data_file,
     machine = os.environ['machine']
     QUEUESERV = os.environ['QUEUESERV']
     ACCOUNT = os.environ['ACCOUNT']
+    CLUSTERS_DTN = os.environ['CLUSTERS_DTN']
+    PARTITION_DTN = os.environ['PARTITION_DTN']
```

Comment:
`CLUSTERS_DTN` and `PARTITION_DTN` may not be defined in all cases, so a default value should be available:

```suggestion
    CLUSTERS_DTN = os.environ.get('CLUSTERS_DTN', None)
    PARTITION_DTN = os.environ.get('PARTITION_DTN', None)
```